### ISSUE TITLE
Can't change borderRadius in outlined input

### ISSUE THREAD

**Issue by BrickmanRodrigo** [n4kz/react-native-material-textfield#291]
Hello, I need to set the borderRadius to 8, but I've seen in the [source code](https://github.com/n4kz/react-native-material-textfield/blob/master/src/components/outline/styles.js#L3) that this is hardcoded as 4

Tried to change this to 8 in any styling way according to the README, but it doesn't work.

```js
<OutlinedTextField
                affixTextStyle={{borderRadius:8}} 
                containerStyle={{borderRadius:8}} 
                inputContainerStyle={{borderRadius:8}} 
                labelTextStyle={{borderRadius:8}} 
                titleTextStyle={{borderRadius:8}} 
                label="Input"
/>
```

Thanks in advance!

**Comment by Harpreet04** [comment]
Did you find any solution for adding borderRadius to outer border?

**Comment by BrickmanRodrigo** [comment]
> 
> 
> Did you find any solution for adding borderRadius to outer border?

No, didn't have any luck yet, tried to mess with the source files and changing the hardcode to 8 but it didn't look right in the corners and the label going up also broke.

**Comment by Harpreet04** [comment]
exactly I tried the same way, I set the default lineWidth prop to zero and customised the border via containerStyle prop, but the floating label( which is the essential transition to keep) is coming over border which messes the design.

**Comment by patelgaurav4u** [comment]
any one has found the solution?
